BlueprintUI/Sources/Element/ElementContent.swift

```diff
@@ -15,20 +15,43 @@ public struct ElementContent: Measurable {
         self.storage = builder
     }
 
-    public func measure(in constraint: SizeConstraint) -> CGSize {
-        return storage.measure(in: constraint)
+    public func measure(in constraint: SizeConstraint, environment: Environment) -> CGSize {
+        return storage.measure(in: constraint, environment: environment)
     }
 
     public var childCount: Int {
         return storage.childCount
     }
 
-    func performLayout(attributes: LayoutAttributes) -> [(identifier: ElementIdentifier, node: LayoutResultNode)] {
-        return storage.performLayout(attributes: attributes)
+    func performLayout(attributes: LayoutAttributes, environment: Environment) -> [(identifier: ElementIdentifier, node: LayoutResultNode)] {
+        return storage.performLayout(attributes: attributes, environment: environment)
     }
 
 }
 
+extension ElementContent {
```

Comment:
Should we also add a companion to some of the ElementContent extensions, like `public init(measureFunction: @escaping (SizeConstraint) -> CGSize)`, but that also pass the env? (Sorry if I missed that in here)

Reply:
Thinking of a case where you may want to vary the measurement based on something in the environment, but I guess we can add that later if it's needed.